boards/DCM/Test/Src/Test_StateMachine.cpp

```diff
@@ -580,4 +648,150 @@ TEST_F(
         0.0f, App_CanTx_GetPeriodicSignal_TORQUE_REQUEST(can_tx_interface));
 }
 
+// DCM21
+TEST_F(DcmStateMachineTest, inverter_switches_closed_when_AIRS_closed)
+{
+    // Both inverter switches are open to begin with
+    LetTimePass(state_machine, 10);
+    ASSERT_FALSE(App_InverterSwitches_IsRightOn(inverter_switches));
+    ASSERT_FALSE(App_InverterSwitches_IsLeftOn(inverter_switches));
+    ASSERT_EQ(turn_on_right_inverter_fake.call_count, 0);
+    ASSERT_EQ(turn_on_left_inverter_fake.call_count, 0);
```

Comment:
```suggestion

```

Reply:
likewise for 667 668

Reply:
Left in
```
ASSERT_EQ(turn_on_right_inverter_fake.call_count, 0);
ASSERT_EQ(turn_on_left_inverter_fake.call_count, 0);
```
and took out the assertions checking if the left and right inverters are on since they depend on HAL_GPIO_ReadPin()